bot.js

```diff
@@ -236,42 +235,29 @@ const redditInterval = async () => {
     const channel = await client.channels.get(REDDIT_POSTING_CHANNEL_ID);
     channel.send(validPost);
   } catch(error) {
+    console.error("Failed to fetch and post Reddit post on interval");
     console.error(error);
   } 
 }
 
 
 
 
-
-
 const limitedMessageHandler = RateLimiter(COMMAND_COOLDOWN, msgHandler);
 client.on("message", limitedMessageHandler);
 
 client.on("guildMemberUpdate", memberUpdateHandler);
 
 
-// Every hour check if the current time is within the ranges
-setInterval(redditInterval, 60 * 60 * 1000);
```

Comment:
This should stay here next to the listeners (client.on('message'), client.on('guildMemberUpdate'), as it is an *invocation point* that starts code processes.